rails-commenting.rb

```diff
@@ -6,39 +6,39 @@
 
 # FILE: app/controller/blog_posts_controller.rb
 
-# ---1)
+# ---1) This line is defining a class called BlogPostsController that inherits all qualities of  ApplicationController
 class BlogPostsController < ApplicationController
   def index
-    # ---2)
+    # ---2) This is an instance variable assigning all blog posts to the variable @posts to display
     @posts = BlogPost.all
   end
 
-  # ---3)
+  # ---3) This is defining a method called show, which will display or read whatever it's assigned to display
   def show
     @post = BlogPost.find(params[:id])
   end
 
-  # ---4)
+  # ---4) This method will create a new object, like a blog post.
```

Comment:
Remember that `new` will display a form for the user, not actually create anything